controllers/playersIndexController.js

```diff
@@ -0,0 +1,29 @@
+const db = require('../db');
+
+// GET http://localhost:7000/admin/players
+exports.getIndex = (req, res, next) => {
+    console.log("GET: Hello getPlayersPage")
+    // connect to the database and save the new incoming player
+    db.getDb()
+        .db('softball')
+        // the collction will be created dynamically if it does not exist yet
+        .collection('players')
+        .find(
+            {},
+            {_id: 0, name: 1, email: 1}
+        )
+        .toArray(
+            function(err, result) {
+                if (err) throw err;
```

Comment:
"throw" is used to crash the app. Weird, right? :) 
So when we use `throw`, there must be `try/catch` somewhere. If there is no `catch` phrase, the app will crash here. 

for example. 
```
try {
  ...long code blah blah
  if(name == null) {
    throw new Error("no-name-in-data");
  }
  ...long code blah blah
}
catch (error) {
  res.status(403).send(...)
}
```
"throw" is like "stop, stop, something is wrong here". When `throw` is executed, the rest of code under `throw` will be skipped. And the process goes into `catch` phrase. It's like "jump" or "escape".
It's helpful when you don't want to think about it too much. 

To summary, 
- `throw` is used with `try / catch` phrase.
- if we can handle our code well and understand everything well, we don't have to use `throw`. 

In my opinion, we don't need `throw` here.

Reply:
Yes, exactly. 👍